.github/workflows/nightly.yml

```diff
@@ -0,0 +1,37 @@
+---
+# A schedule only runs on branches that match include rules from _both_ `main`
+# and the branch itself.  On `main`, we blanket include all branches that might
+# want to be enabled, then particular branches can override it to exclude
+# themselves by removing themselves from the trigger list.  For example, old
+# stable branches can remove `stable/*` from their copy of this file once they
+# reach their end-of-life.
```

Comment:
Is this true? IIRC the schedule trigger only runs on the default branch.

Reply:
This was an old comment, this behavior probably works in Azure but not in GH Actions as it should only run scheduled workflows on the main branch. Removed in 4655adc.